Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM codait/max-base:v1.1.3
 
-ARG model_bucket=http://max-assets.s3.us.cloud-object-storage.appdomain.cloud/image-segmenter/1.0
+ARG model_bucket=https://max-assets-dev.s3.us-south.cloud-object-storage.appdomain.cloud/max-image-segmenter/1.0.0
```

Comment:
can you change it to `max-assets-prod`? I have already uploaded the assets on prod for the other deployment.

Reply:
And is `-south` necessary? I thought we had some discussion on slack...

Reply:
One of them was the old URL. We can't/shouldn't use that anymore.